docs/hosting-a-server-with-northstar/troubleshooting.md

```diff
@@ -31,9 +14,9 @@ Check Northstar's Discord for annoucements.
 
 **Ports are not the same**
 
-Your gameserver is configured to listen to a given TCP port.
+Your gameserver is configured to listen to a given UDP port.
 
-Masterserver needs to be able to contact your gameserver though that same port.
+Masterserver needs to be able to contact your gameserver though that same port (i.e., you can't forward a different external port).
```

Comment:
Again, same `v1.12` vs `v1.13` differentiation.

Reply:
This change is also meant for v1.12; the other sections are about TCP.